### PR TITLE
html_render progress

### PR DESCRIPTION
**Pull request by spacemollusk**
This bit contains commits for html_render progress.

### CODE REVIEW

**Comment by spacemollusk** [comment]
Completed circle.py code (but not full tests or optional features yet).

File: students/jim/Lesson03/mailroom.py [hunk 1]
```diff
@@ -70,5 +65,14 @@ def create_report():
...
 
+
+def print_letter(donor_name, donation_amt):
+    print("Dear {}:\n".format(donor_name))
+    print("Thank you so much for your kind donation of {} dollars.".format(donation_amt))
+    print("It will be put to great use. Give yourself a big pat on the back.\n")
+    print("Sincerely,")
+    print("The Management.")
+    return
+
```
Comment (summerela) [inline students/jim/Lesson03/mailroom.py]: 👍

Comment (summerela) [reply students/jim/Lesson03/mailroom.py]: You could also wrap this into a raw docstring: 

    r'''Dear {}:
    Now I am printing some stuff...
    And more stuff...
    '''

File: students/jim/Lesson08/circle.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    def __add__(*args):
+        rad_sum = 0
+        for arg in args:
+            if type(arg) is not Circle:
+                raise TypeError("Incompatible operand:", arg)
+                break
+            else:
+                rad_sum += arg.rad
+        return Circle(rad_sum)
+
```
Comment (summerela) [inline students/jim/Lesson08/circle.py]: 👍

**Comment by summerela** [comment]
Nicely done! I only had a few small suggestions.

**Pull request merged**